mozilla/blurts-server
# MNTOR-2245 - pass Glean config from env var in server component to cl…

…ient component



# References: 
Jira: MNTOR-2245



# Description

Read Glean config from env var in server component, and pass to client component.

I split this out from the other places in the code where we encounter https://mozilla-hub.atlassian.net/browse/MNTOR-2237 since this is blocking Nimbus testing and is relatively simple.

# How to test

Set `APP_ENV` to different values and observe different Glean channel pings being sent.

# Checklist (Definition of Done)
